worker-build/src/js/shim.js

```diff
@@ -20,6 +20,12 @@ class Entrypoint extends WorkerEntrypoint {
     async scheduled(event) {
         return await imports.scheduled(event, this.env, this.ctx)
     }
+
+    // For some reason, email events doesn't seem to use WorkerEntrypoint so we get the env and ctx from
+    //  from the function itself.
+    async email(message, _env, _ctx) {
```

Comment:
@ObsidianMinor do you know why email worker wouldn't use the new interface? Will cron triggers have the same issue?

Reply:
I still haven't figured it out yet - for some reason `env` is still not probably propagated into the rust layer, and therefore I can't seem to use bindings with it :sweat_smile:

Reply:
It also quite difficult to debug because email workers can't be run locally :sweat_smile: